test.py

```diff
@@ -265,41 +86,46 @@ def inference_args(parser, remaining_args):
         help='disables CUDA inference'
     )
 
+    torch.autograd.set_detect_anomaly(True)
```

Comment:
what is this for?

Reply:
?

Reply:
just debugging backprop. feel free to remove the 'gradient' flag there

Reply:
addressed in 0cd1cab